ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -17,19 +17,34 @@ class Tag {
 export default class FormWorkerCreateWorkerLedComponent extends Component {
   // =services
   @service features;
+  @service browserObject;
 
   // =attributes
   @tracked generatedWorkerAuthToken;
-  @tracked clusterId;
+  @tracked clusterId = this.clusterIdFromURL;
   @tracked ipAddress;
   @tracked configFilePath;
   @tracked initialUpstreams;
   @tracked workerTags = A([]);
   @tracked newWorkerKey;
   @tracked newWorkerValue;
 
-  // =properties
+  get clusterIdFromURL() {
```

Comment:
```suggestion
  get clusterIDFromURL() {
```

Reply:
I'll update this but it looks like we're pretty inconsistent about `Id` vs `ID` as part of camel casing, should we standardize?